pages/learn/deploy/delta.md

```diff
@@ -8,30 +8,40 @@ thumbnail: /img/runtime/ResinSupervisorDelta.png
+__Note__: Delta updates are enabled by default for devices running {{ $names.os.lower }} >= 2.47.1 or [ESR](esr) versions >= 2020.04
+
+For any devices running {{ $names.os.lower }} >= 2.47.1, the delta update behavior is enabled by default. For devices running {{ $names.os.lower }} < 2.47.1, updating to >= 2.47.1 via a [self-service update][self-service-update] will enable delta updates for the device. Alternatively, the delta update behavior may be enabled or disabled application-wide or per-device with the `RESIN_SUPERVISOR_DELTA` configuration variable.
 
 ![Setting the fleet configuration to enable delta behavior](/img/runtime/ResinSupervisorDelta.png)
 
 To enable this behavior application-wide, that is for all devices of a given application, set the above variable at **Fleet Configuration** in the {{ $names.company.lower }} dashboard of your application, through the {{ $names.company.lower }} [API](/reference/api/resources/application_config_variable/), through the SDK (in [Node.js](/reference/sdk/node-sdk/#configvar-set-nameorid-key-value-code-promise-code-) or [Python](/reference/sdk/python-sdk/#applicationconfigvariable)), or the [command line interface](/tools/cli/#envs).
 
 To enable this behavior on a per-device basis, set the above variable at **Device Configuration** in the {{ $names.company.lower }} dashboard for your device, through the {{ $names.company.lower }} [API](/runtime/data-api/#create-device-variable), through the SDK (in [Node.js](/reference/sdk/node-sdk/#configvar-set-uuidorid-key-value-code-promise-code-) or [Python](/reference/sdk/python-sdk/#deviceconfigvariable)), or the [command line interface](/tools/cli/#envs). If the device is [moved to another application](/management/devices/#move-to-another-application), it will keep the delta updates behavior regardless of the application setting.
 
+Similarly, you may disable the delta update behavior application-wide or per-device.
+
 ## Delta behavior
 
-If you are using delta updates, you might notice the following changes in {{ $names.company.lower }} behavior:
+Before devices can update using deltas, a delta must be generated between the release a device is currently running and the one it is updating to. There are two ways deltas are generated:
+
+1. Automatically between the last successful and the new release during a build on the {{$names.cloud.lower}} builder - these are known as "build-time" deltas.
+2. Automatically when a device is requesting to update from or to a release for which no delta has been generated before - these are known as "on-demand" deltas and can typically happen if the device was not on the latest release when the new release was pushed, or the user did not use the {{$names.cloud.lower}} builder for the new release.
+
+__Note:__ Delta updates that were enabled for applications created prior to December 2019 (v2) do not generate build-time deltas, and will always generate an on-demand delta.
```

Comment:
This is not telling the full story and also feels redundant, especially having to mention a specific date.